tests/integration/post.hurl

```diff
@@ -0,0 +1,35 @@
+# Test POST /users endpoint
+
+# 1. Create new user (should succeed)
+POST http://localhost:8080/v1/users
+[BasicAuth]
+user1@email.com:password1
+HTTP 201
+[Asserts]
+jsonpath "$.status" == "ok"
```

Comment:
Let's not use `response.js`. The `status` property in the body is redundant when we have HTTP status codes.